src/org/traccar/database/AttributesManager.java

```diff
@@ -26,9 +26,14 @@ public AttributesManager(DataManager dataManager) {
     }
 
     @Override
+    public Attribute getById(long calendarId) {
```

Comment:
It's really easy:

https://docs.oracle.com/javase/tutorial/java/generics/types.html